src/collections/blog/2020-08-11-my-journey-from-a-contributor-to-a-maintainer/index.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: "My journey from a contributor to a maintainer!"
+subtitle: ""
+date: 2020-08-11 10:55:05 -0530
+author: Nikhil Ladha
+thumbnail: ./pic1.png
+tags:
+ - Community
+ - Projects
+ - News
+published: true
+---
+
+import { BlogWrapper } from "../Blog.style.js";
+import pic1 from "./pic1.png";
+import pic2 from "./pic2.jpeg";
+
+<BlogWrapper>
+
+Getting involved in something new and essentially avant garde, can be a messy challenge to navigate through. But, when it comes to contributing to Meshery it only gets more and more meshy!
+<img src={pic1} class="image-right"/>
+#### Get set go!
```

Comment:
This is rendered as it is and not as a heading. Please retry doing this.

Reply:
Do I need to remove this? As the same image is repeated here.

Reply:
Yes. Remove the image. Also leaving a line between the heading and the paragraph will fix the rendering as it issue.